### ISSUE TITLE
Need an argument in fwrite to force writing a decimal point to numeric columns

### ISSUE THREAD

**Issue by renkun-ken** [Rdatatable/data.table#2864]
One downside of CSV format is that the columns do not preserve types. A numeric column that happens to be fully represented by integers is written as integer in CSV. Next time the file is read, the column becomes `integer` unless `colClasses` is specified to force reading the column as numeric. 

I'm wondering if it makes sense to provide an argument in `fwrite` to force numeric columns to end up with decimal point or numbers (e.g. `1` is written as `1.0`) so that next time the file is read, the type of the column is still numeric.

**Comment by MichaelChirico** [comment]
of course you can sprintf('%.1f') your columns, I guess you want an option
for this to be done for all columns stored as numeric?

**Comment by renkun-ken** [comment]
Yes, that's exactly what I'm trying to say. I think it makes sens that `fread()` restores as much as possible what's done by `fwrite()`
